qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data
+  file = File.new('data.txt', 'r:UTF-8')
+  file.read
+end
+
+def weather_data(town, data)
+  monthly_rainfall = data.split("\n").find { |data_row| data_row.match "#{town}:" }
+  return monthly_rainfall.scan(/\d*\.\d/).map(&:to_f) unless monthly_rainfall.nil?
+end
+
+def mean(town, data)
+  rainfall_values = weather_data(town, data)
+  return -1.0 unless data.match "#{town}:"
```

Comment:
the whole point of the line above is to let you check the `rainfall_values` instead of the redundant `match` operation 😄